test/functionalTest/cases/3442_ngsiv1_interop_location_fix/ngsiv1_geobox_create_does_not_break_location.test

```diff
@@ -39,18 +39,18 @@ echo "01. Create entity with geo:box with NGSIv1 (APPEND)"
 echo "==================================================="
 payload='{
   "contextElements": [
-  {
-    "type": "E",
-    "isPattern": "false",
-    "id": "E1",
-    "attributes": [
     {
-      "name": "position",
-      "type": "geo:box",
-      "value": ["7.7, 8.8", "9.9, -10.10"]
+      "type": "E",
+      "isPattern": "false",
+      "id": "E1",
+      "attributes": [
+        {
+          "name": "position",
+          "type": "geo:box",
+          "value": ["7.7, 8.8", "9.9, -10.10"]
+        }
+      ]
```

Comment:
Why this change? Indent only?

Reply:
Yeah, it hurt my eyes. Sorry

Reply:
It makes review a little harder ;) but it's fine. NTC.